userspace/falco/configuration.cpp

```diff
@@ -148,7 +148,23 @@ void falco_configuration::init(string conf_filename, list<string> &cmdline_optio
 		m_outputs.push_back(http_output);
 	}
 
-	if (m_outputs.size() == 0)
+	m_grpc_enabled = m_config->get_scalar<bool>("grpc", "enabled", false);
+	m_grpc_bind_address = m_config->get_scalar<string>("grpc", "bind_address", "0.0.0.0:5060");
+	m_grpc_threadiness = m_config->get_scalar<uint32_t>("grpc", "threadiness", 8); // todo > limit it to avoid overshubscription? std::thread::hardware_concurrency()
```

Comment:
Investigate threads oversubscription

Reply:
I think we should parse this dynamically from `/proc`?

Reply:
I think that they are not exactly the same thing. There are various things we should consider reading this info from `/proc` - eg., hyperthreading enabled or not, processors turned off.

Thus, at maximum I would consider using `/proc/cpuinfo` processors count only as fallback when `hardware_concurrency()` returns 0 (which means not implemented).